### PR TITLE
Fixes Issue #625: Remove breadcrumb margins

### PR DESCRIPTION
**Pull request by cpoff**
[Original issue](https://github.com/az-digital/arizona-bootstrap/issues/625)

This eliminates all margins around the breadcrumbs. Two files impacted. One (`arizona-bootstrap.css`) had explicit values that are now set to 0. The second (`_breadcrumb.scss`) referenced variables contained in `_custom-variables.scss` where I also re-added some removed breadcrumb styles and set to 0.

I expect some build errors on this initial change.

### CODE REVIEW

**Comment by mmunro-ltrr** [comment]
The CSS files under dist/css are actually derived, so there's no need to edit them directly (but the minified version from there should be changing, too).

**Comment by trackleft** [comment]
As @mmunro-ltrr said, you should be able to add those variables here: https://github.com/az-digital/arizona-bootstrap/blob/main/scss/_custom-variables.scss#L231

**Comment by cpoff** [comment]
Got 'em. Did some fat-fingered commit commands, so I'm trying to clean up my self-made mess. Should have it all pushed shortly.

**Comment by mmunro-ltrr** [comment]
I think there's a problem with just removing the .breadcrumb padding & margin-bottom from Arizona Bootstrap, since then the Twitter Bootstrap defaults come through, without any overrides to prevent this. The demo content to examine is at
https://review.digital.arizona.edu/arizona-bootstrap/issue/remove-breadcrumb-padding-625/docs/2.0/components/breadcrumb/

File: scss/_breadcrumb.scss [hunk 1]
```diff
@@ -2,8 +2,6 @@
 .breadcrumb {
   display: flex;
   flex-wrap: wrap;
-  padding: $breadcrumb-padding-y $breadcrumb-padding-x;
-  margin-bottom: $breadcrumb-margin-bottom;
```
Comment (mmunro-ltrr) [inline scss/_breadcrumb.scss:6]: Restore these lines.

Comment (mmunro-ltrr) [reply scss/_breadcrumb.scss:6]: In `_custom-variables.scss`, modify the breadcrumbs section to look like
```scss
// >> Breadcrumbs
$breadcrumb-color: $azurite;
$breadcrumb-bg: $white;
$breadcrumb-padding-y: 0px;
$breadcrumb-padding-x: 0px;
$breadcrumb-margin-bottom: 0px;
```

Comment (cpoff) [reply scss/_breadcrumb.scss]:
```
scss/_custom-variables.scss
 233:25  ✖  Unexpected unit  length-zero-no-unit
 234:25  ✖  Unexpected unit  length-zero-no-unit
 235:29  ✖  Unexpected unit  length-zero-no-unit
```

**Review by mmunro-ltrr — APPROVED** [review]
Looks OK on review site.

**Review by trackleft — APPROVED** [review]

**Comment by trackleft** [comment]
FYI this might warrant a minor release.

File: scss/_breadcrumb.scss [hunk 2]
```diff
@@ -35,3 +35,12 @@
   color: $breadcrumb-divider-color;
   content: "chevron_right";
 }
+
+// Remove margins/padding on Breadcrumbs for larger displays
+@include media-breakpoint-up(lg) {
+  .breadcdrumb {
```
Comment (trackleft) [inline scss/_breadcrumb.scss:41]: Nitpick
```suggestion
  .breadcrumb {
```

**Review by mmunro-ltrr — APPROVED** [review]

**Pull request merged**